src/lib.rs

```diff
@@ -164,6 +166,7 @@ pub fn middleware(app: Arc<App>) -> MiddlewareBuilder {
         "/crates/:crate_id/reverse_dependencies",
         C(krate::reverse_dependencies),
     );
+    // Used to generate download graphs
```

Comment:
I might have led you astray, and I just checked to be sure-- I think this comment should go with the `/crates/:crate_id/downloads` route defined [up here](https://github.com/rust-lang/crates.io/pull/936/files#diff-b4aea3e418ccdb71239b96952d9cddb6L151).